src/prefect/cli/deploy.py

```diff
@@ -435,7 +438,7 @@ async def _run_single_deploy(
         prefect_dir = find_prefect_directory()
         if not prefect_dir:
             raise ValueError(
-                "No .prefect directory could be found - run [yellow]`prefect project"
+                "No .prefect directory could be found - run [yellow]`prefect"
```

Comment:
It won't create one anymore though. Should we mention deprecation here?

Reply:
I tested it, and `prefect init` still created a `.prefect` folder.

Reply:
Oh I see, I guess I'll work on that one later.